components/Dialogs/src/MDCAlertController.h

```diff
@@ -88,6 +88,9 @@
 /** The color applied to the button ink effect of Alert Controller.*/
 @property(nonatomic, strong, nullable) UIColor *buttonInkColor;
 
+/** The corner radius applied to the Alert Controller shape. Default to 0 (no round corners) */
```

Comment:
Should this be "Alert Controller **view**" instead of "shape"?  It doesn't look like the property is being applied to any Shaping scheme objects.